src/Parser/Mage/Fire/Modules/Features/Abilities.js

```diff
@@ -73,26 +90,22 @@ class Abilities extends CoreAbilities {
       {
         spell: SPELLS.FLAMESTRIKE,
         category: Abilities.SPELL_CATEGORIES.ROTATIONAL_AOE,
+        isOnGCD: true,
+        castEfficiency: {
```

Comment:
it is disabled by default so no need to include that

Reply:
its only disabled by default if you dont list a cooldown. If a cooldown is set, then its enabled by default.

That said, ill remove it from the ones where there is no cooldown set.